docs/FunctionalTestBuildScripts.md

```diff
@@ -0,0 +1,13 @@
+# Build Scripts
+
+This file records scripts used to compile the test files, by alphabetical order.
```

Comment:
>by alphabetical order [](http://example.com/codeflow?start=58&length=21)

either:
`in alphabetical order.` or
`ordered alphabetically.`

Unfortunately, this is one of those phrases that doesn't translate well between most languages and the preposition choice can seem arbitrary. The first option tends to be the preferred phrasing in American English. The second translates a little easier for some languages.

Reply:
fixed using in.